### DISCUSSION TITLE
Difficult Adjust Score x0.5 by Ar10 ?

### DISCUSSION THREAD

**Discussion by OsuGerman** [ppy/osu#19988] [Ideas]
I think high Ar should also be considered when ajusting the map.

Now it dosent matter what i change the score multi is 0.5x

**Comment by mk56-spn** [comment]
This is a temporary patch to stop CS1 leaderboard cheese and the likes . Things like that will come later.

**Reply by OsuGerman** [reply]
okay great

**Reply by OsuGerman** [reply]
![osu_2022-08-27_22-33-03](https://user-images.githubusercontent.com/89344482/187048436-32f2f13e-dca4-4a33-9117-c288edb38014.jpg)
![osu_2022-08-27_22-52-20](https://user-images.githubusercontent.com/89344482/187048439-32303533-2398-4245-940a-f53f328a5ed0.jpg)

is that why after my PC restart my Score is not that good anymore ?

**Reply by mk56-spn** [reply]
mustve auto updated the lazer version, or some other funky score processing delay im not aware of , eitherway theres no point worrying about this, it will be adressed in due time
